pyproject.toml

```
@@ -1,57 +1,62 @@
 [tool.poetry]
 name = "vast-pipeline"
-version = "v1.2.0-dev"
+version = "2.0.0dev"
 description = "Vast Pipeline code base for processing and analysing telescope images from the Square Kilometre Pathfinder"
 authors = ["The VAST Development Team"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.8.0,<3.11"
-astropy = "^5.0"
-astroquery = "^0.4.4"
-bokeh = "2.4.2" # must align with @bokeh/bokehjs version in package.json
-cloudpickle = "^1.5.0"
-dask = {extras = ["dataframe"], version = "^2022.1.0,<2022.4.2"}
-dill = "^0.3.3"
-Django = "^3.2.13"
-django-crispy-forms = "^1.9.2"
-django-environ = "^0.4.5"
-djangorestframework = "^3.11.0"
-djangorestframework-datatables = "0.5.1"
-django-q = "^1.3.3"
-django-tagulous = "^1.1.0"
-Jinja2 = "^3.0.3"
-networkx = "^2.4"
-numpy = ">=1.18.1,<1.23"
-pandas = "^1.2.0"
-psutil = "^5.7.0"
-psycopg2 = "^2.8.4"
-pyarrow = "^7.0"
-scipy = "^1.6.0"
-social-auth-app-django = "^3.1.0"
-social-auth-core = "^3.3.3"
-sqlalchemy = "^1.3.11"
-whitenoise = "^5.2.0"
-gevent = { version = "^21.1.2", optional = true }
-gunicorn = { version = "^20.0.4", optional = true }
-forced-phot = { git = "https://github.com/askap-vast/forced_phot.git", tag = "v0.2.0"}
-strictyaml = "^1.6.2"
+python = ">=3.11.0,<4"
```

I'm not sure what best practice is, but in my head this should be set to whatever python versions we actually test, i.e. something like this

```suggestion
python = ">=3.11.0,<3.13"
```